cli/command/builder/prune.go

```diff
@@ -9,6 +9,7 @@ import (
 	"github.com/docker/cli/cli"
 	"github.com/docker/cli/cli/command"
 	"github.com/docker/cli/cli/command/completion"
+	"github.com/docker/cli/internal/prompt"
```

Comment:
could we have it in `pkg` so we can use it in buildx repo (https://github.com/docker/buildx/blob/646df6d4a06848c675abf110dd51c2623b517590/commands/util.go#L15-L57)?

Reply:
Ah, right, but that would probably not work currently? i.e., that fork was because of https://github.com/docker/buildx/pull/2359#discussion_r1544736494, or was that part of it reverted?

My thinking of keeping it "internal" for now at least, was because there may still be some work to do to get a stable "API" for these. There's some functions that currently depend on `command.Stream` (or equivalent), and _ideally_ they'd be more generic (I'm a bit on the fence on the `Stream` wrappers; they're useful, but perhaps we can do without so that we can pass a regular `os.StdIn` / `os.StdErr` etc).